core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -473,8 +473,8 @@ object LogConfig {
     FlushMessagesProp -> asList(
       new ConfigSynonym(KafkaConfig.LogFlushIntervalMessagesProp)),
     FlushMsProp -> asList(
-      new ConfigSynonym(KafkaConfig.LogFlushSchedulerIntervalMsProp),
-      new ConfigSynonym(KafkaConfig.LogFlushIntervalMsProp)),
+      new ConfigSynonym(KafkaConfig.LogFlushIntervalMsProp),
+      new ConfigSynonym(KafkaConfig.LogFlushSchedulerIntervalMsProp)),
```

Comment:
Is this the fix for KAFKA-13807? Did we have the precedence wrong or something?

Reply:
yes, this is the fix for KAFKA-13807